client/mass/groups.js

```diff
@@ -302,13 +302,18 @@ async function updateUI(self) {
 		columns: [
 			{
 				label: 'NAME',
-				editCallback: (i, cell) => {
-					// group name is changed
-					groups[i].name = cell.value
-					self.app.dispatch({
-						type: 'app_refresh',
-						state: { groups }
-					})
+				editCallback: async (i, cell) => {
+					const newName = cell.value
+					const index = self.state.groups.findIndex(group => group.name == newName)
```

Comment:
I think this check should be done in the `mass/store.js`, in case other places allow group name edits, then no need to duplicate this validation check

Reply:
can look at it later